src/Controller/AdminController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\AdminManager;
+use App\Model\SessionHandler;
+
+class AdminController extends AbstractController
+{
+    public function password(): string
+    {
+        return $this->twig->render('Admin/password.html.twig');
+    }
+
+    public function checking(): string
+    {
+        $info = new AdminManager();
+        $info = $info->selectInfoAdmin();
+
+        if ($_POST['password'] === $info['pass']) {
```

Comment:
You must not store user password
You need to hash password before storing them in the database using php's password_hash function
And user password_verify() to check if the provided password match the stored hash